### ISSUE TITLE
Project not updating via update.json

### ISSUE THREAD

**Issue by coreybutler** [jsdelivr/jsdelivr#12519]
I have the following in the [chassis](https://github.com/jsdelivr/jsdelivr/tree/master/files/chassis) project's update file:

```
{
  "packageManager": "npm",
  "name": "ngn-chassis",
  "repo": "ngnjs/cdn",
  "files": {
    "basePath": "chassis-lib/",
    "include": ["*.min.js"]
  }
}
```

**Expected Behavior**

When the bot detects a new version of ngn-chassis in npm, it triggers an update. The update looks at the `ngnjs/cdn` repository and pulls all files ending in `.min.js` from the `chassis-lib` directory.

The resulting output is stored in a directory whose name corresponds to the version npm module's version number.

**Actual Behavior**

Nothing. It's been 3 days with this configuration and it still shows version 1.0.56 even though there have been multiple incremental updates each day (currently at v1.0.63 as of this writing).

---

So, the big questions are

a) Am I understanding the process correctly?
b) Why aren't the updates happening?
c) Could the [info.ini](https://github.com/jsdelivr/jsdelivr/blob/master/files/chassis/info.ini) file's `github` property be confusing the process? 

---

Further information | Use Case: 

This used to be triggered by a Github release, but not any longer. We _used_ to have a `dist` directory in the chassis-lib repo. This became a problem for our workflow... plus, it's just not good practice to have your release files stored with your source code. SCM is suppposed to be about _source_ code management, not release code.

I have since configured my build process to generate the appropriate release files as a Github release. This triggers an external process that commits those files to the [ngnjs/cdn](https://github.com/ngnjs/cdn) repository. Upon completion of that, my build process then publishes to npm.

Basically, I'm using npm to trigger the jsdelivr bot, but using the ngnjs/cdn repo as the source for the bot.

I posted a comment on a [similar issue](https://github.com/jsdelivr/libgrabber/issues/51) in the libgrabber repo and offered to help out with it.

**Comment by coreybutler** [comment]
Bump

**Comment by MartinKolarik** [comment]
You can't use npm to "trigger the bot" and GitHub repo to download the files. You either have the code on GitHub and use GitHub releases, or use npm and have the code there.

If that makes sense, add the files you want on the CDN to your npm package. If not, keep them in to repo, change `update.json` to use GitHub, and tag the releases in the `ngnjs/cdn` repo.

**Comment by coreybutler** [comment]
@MartinKolarik - This makes sense and I now understand why things aren't working.

Having to put ALL of the library code into an npm package can be a pretty poor practice.  My library consists of individual files, plus a concatenated version (for people who want a one-liner in the `<head>`, plus sourcemaps. That's alot of extra unnecessary code shipping to npm, which can become an issue (I have some users hosting stuff on Raspberry Pi devices with limited disk capacity... every bit helps). I wrote a piece on "why" this is bad some time ago in [NPM Needs a Personal Trainer](https://medium.com/@goldglovecb/npm-needs-a-personal-trainer-537e0f8859c6#.tz0j4wt8r).

Again, I'm willing to help out... particularly in deploying files from a Github release (as opposed to the source repo). Is this something you'd be open to?

**Comment by MartinKolarik** [comment]
As I said, your second option (and I agree this is the better one) is using another repo for CDN code. You already have the repo, you just need to tag releases in `ngnjs/cdn` (like you do in `ngnjs/chassis-lib`).

**Comment by coreybutler** [comment]
That CDN repo contains 3 separate projects.... if I setup a release, I believe I'd be triggering all 3 projects to look for a new version.

**Comment by MartinKolarik** [comment]
Oh I see... In that case, there's one more option... You can have the release files in the original repo, but on a different branch. E.g., you can have a branch "builds" that contains the release files and make GitHub releases from that branch instead of "master".

**Comment by coreybutler** [comment]
That could work... I'll play around with that. 

The offer still stands if you want anyone to help contribute something to pull git release files though.

**Comment by MartinKolarik** [comment]
>The offer still stands if you want anyone to help contribute something to pull git release files though.

Thanks but I think the existing options are easier to use in most scenarios (maybe even in this one).

**Comment by jimaek** [comment]
The new backend is live. It pulls files directly from npm and github https://github.com/jsdelivr/jsdelivr#usage
